### ISSUE TITLE
Retrospectives Teams assessments: Row headers should be defined within the table

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/vsts-extension-retrospectives#1064]
## Repro Steps:

1. Open an ADO project
2. Select board > Retrospectives from left navigation bar
3. Activate Team assessment button
4. Turn on Narrator or other screen reader.
5. Observe row header is not defined.

## Actual Result.

The row header should be defined for the table to ensure proper accessibility for screen reader users.

![Image](https://github.com/user-attachments/assets/7902ea0e-d377-45de-8dfb-c3a0107fdf88)

**Comment by msftedad** [comment]
#A11y #accessibility issue
